defaults/main.yml

```diff
@@ -45,3 +46,5 @@ prom_temp_dir: "%TEMP%"
 prom_app_dir: C:\Program Files\prometheus
 prom_data_dir: "%ProgramData%\\prometheus"
 prom_config_file: "{{ prom_data_dir }}\\prometheus.yml"
+
+localdownload: "false"
```

Comment:
To be consistent with other vars, can we make this a boolean?
```
localdownload: false
```

Reply:
I have corrected all localdownload and, per [this link](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_conditionals.html#conditionals-based-on-variables), added with `| bool`
Let me know how you see it

Reply:
The filter is fine to include. It'll catch a few more cases.